QueryBuilder.Test/Models/Extensions.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
I feel like what might be more appropriate is to have the unit test project have a set of json DTDL models and have make use of an AssemblyInitialize method such that prior to any of the actual tests running, when the assembly is spun up, it would regenerate the models using the DTDL model generator, ensuring the tests are using the helper classes that come along with the generator instead of us having to manually copy them here and manually maintain our own test model(s)

Reply:
This would be similar to how the model generator generates its own models for its tests.

Reply:
I've added the DTDL model generator to QueryBuilder.Test. It runs on assembly initialize and creates models in a new QueryBuilder.Test.Generated csproj.